scripts/build/Platform/Windows/build_config.json

```diff
@@ -366,11 +362,32 @@
         "CMAKE_OPTIONS": "-G \"Visual Studio 16 2019\" -DCMAKE_SYSTEM_VERSION=10.0 -DLY_DISABLE_TEST_MODULES=TRUE -DLY_VERSION_ENGINE_NAME=o3de-sdk -DLY_INSTALLER_WIX_ROOT=\"!WIX! \"",
         "EXTRA_CMAKE_OPTIONS": "-DLY_INSTALLER_AUTO_GEN_TAG=ON -DLY_INSTALLER_DOWNLOAD_URL=!INSTALLER_DOWNLOAD_URL! -DLY_INSTALLER_LICENSE_URL=!INSTALLER_DOWNLOAD_URL!/license",
         "CPACK_BUCKET": "!INSTALLER_BUCKET!",
-        "CMAKE_LY_PROJECTS": "",
         "CMAKE_TARGET": "ALL_BUILD",
         "CMAKE_NATIVE_BUILD_ARGS": "/m /nologo"
     }
   },
+  "install_profile_vs2019_pipe": {
+    "TAGS": [
+      "nightly-incremental",
+      "nightly-clean"
+    ],
+    "PIPELINE_ENV": {
+      "PROJECT_REPOSITORY_NAME": "TestProject"
+    },
+    "steps": [
+      "install_profile_vs2019",
+      "project_generate",
+      "project_engineinstall_profile_vs2019"
+    ]
+  },
+  "project_generate": {
+    "TAGS": [],
+    "COMMAND": "python_windows.cmd",
+    "PARAMETERS": {
+      "SCRIPT_PATH": "install\\scripts\\o3de.py",
+      "SCRIPT_PARAMETERS": "create-project -pp %WORKSPACE%\\!PROJECT_REPOSITORY_NAME! --force"
```

Comment:
Shouldn't all the WORKSPACE references be `%WORKSPACE%` since it is set before the `*.cmd` scripts are running?

I see some were added that that uses `!WORKSPACE!` and that should be needed

Reply:
yeah, all of them should be set with %, will change, I think `PROJECT_REPOSITORY_NAME` can also be with %% since that is not set within the cmd